code/addons/test/src/vitest-plugin/viewports.ts

```diff
@@ -47,8 +57,18 @@ const parseDimension = (value: string, dimension: 'width' | 'height') => {
   }
 };
 
-export const setViewport = async (viewportsParam: ViewportsParam = {} as ViewportsParam) => {
-  const defaultViewport = viewportsParam.defaultViewport;
+export const setViewport = async (parameters: Parameters = {}, globals: Globals = {}) => {
+  let defaultViewport;
+  const viewportsParam: ViewportsParam = parameters.viewport ?? {};
+  const viewportsGlobal: ViewportsGlobal = globals.viewport ?? {};
+  const isDisabled = viewportsParam.disable || viewportsParam.disabled;
```

Comment:
should this take `viewportsGlobal.disable` into account too?

Reply:
From what I can see there is no such thing as global.viewport.disable:
https://github.com/storybookjs/storybook/blob/1dadd476f9bd3ab73880bd2afb77ecd6aaed7cb0/code/addons/viewport/src/components/Tool.tsx#L42

can you show me where it's used?